src/include/wiredtiger.in

```diff
@@ -5620,132 +5620,130 @@ extern int wiredtiger_extension_terminate(WT_CONNECTION *connection);
 #define	WT_STAT_CONN_CHILD_MODIFY_BLOCKED_PAGE		1355
 /*! transaction: Number of prepared updates */
 #define	WT_STAT_CONN_TXN_PREPARED_UPDATES_COUNT		1356
-/*! transaction: Number of prepared updates added to cache overflow */
-#define	WT_STAT_CONN_TXN_PREPARED_UPDATES_LOOKASIDE_INSERTS	1357
 /*! transaction: durable timestamp queue entries walked */
-#define	WT_STAT_CONN_TXN_DURABLE_QUEUE_WALKED		1358
+#define	WT_STAT_CONN_TXN_DURABLE_QUEUE_WALKED		1357
```

Comment:
I'm not sure what's the side effect of this. :( May be we cannot remove a statistic.

Reply:
We can remove statistics.